Blockzilla/URLBar.swift

```diff
@@ -644,6 +644,12 @@ class URLBar: UIView {
     @objc private func didTapShieldIcon() {
         delegate?.urlBarDidTapShield(self)
     }
+    
+    @objc func urlBarDidLongPressUrl(sender: UILongPressGestureRecognizer) {
```

Comment:
Do we want Telemetry on this? It seems like it could be valuable to know how many people even open this menu in the first place. May want to ask Barbara / Susheel about this.

Reply:
Unless I'm missing something, it looks like we haven't historically had Telemetry on this. I'll inquire about whether we want to, but I think it should be a separate PR/issue than the redesign.